beacon-chain/rpc/validator/proposer.go

```diff
@@ -40,12 +40,12 @@ var eth1DataNotification bool
 const eth1dataTimeout = 2 * time.Second
 
 type eth1DataSingleVote struct {
-	eth1Data    ethpb.Eth1Data
+	eth1Data    *ethpb.Eth1Data
 	blockHeight *big.Int
 }
 
 type eth1DataAggregatedVote struct {
-	data  eth1DataSingleVote
+	data  *eth1DataSingleVote
```

Comment:
same here

Reply:
This was due to a change of how protobuf handles structs not being a pointer. If this weren't a pointer protobuf would it would fail compilation.

Reply:
Not sure I understand, what does this have to do with protobufs ? its a normal struct so any protobuf specific gotchas shouldnt apply here.

Reply:
I replicated the problem but it turns out this change was not needed, thank you for pointing it out!